sdk/storage/Azure.Storage.Internal.Avro/src/Azure.Storage.Internal.Avro.csproj

```diff
@@ -6,7 +6,6 @@
     <AssemblyTitle>Microsoft Azure.Storage.Internal.Avro client library</AssemblyTitle>
     <Version>12.0.0-preview.1</Version>
     <GenerateAPIListing>false</GenerateAPIListing>
-    <EnableApiCompat>false</EnableApiCompat>
```

Comment:
Why are you removing these?

Reply:
Got it, controlled with ApiCompatVersion now.

Reply:
Yes.